maxim/models/dataset.py

```
@@ -9,128 +9,213 @@ class VariableType(str):
-            context=context_var,
-            expectedOutput=expected_output_var
-        )
+        Returns:
+            Dict[str, Any]: Dictionary representation with rowNo, columnName, type, and value
+        """
+        # Map internal "file" type to API's expected "attachment" type
+        return {
+            "rowNo": self.row_no,
+            "columnName": self.column_name,
+            "type": self.type,
+            "value": [] if self.type == "file" else self.payload
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)